shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/BridgeIngressServiceImpl.java

```diff
@@ -144,4 +143,36 @@ public void deleteBridgeIngress(BridgeDTO bridgeDTO) {
             LOGGER.warn("BridgeIngress '{}' not deleted", bridgeDTO);
         }
     }
+
+    @Override
+    public void createOrUpdateBridgeIngressSecret(BridgeIngress bridgeIngress, BridgeDTO bridgeDTO) {
```

Comment:
@robpblake it's best practice to store confidential info like credentials/passwords in Secrets. In such way it's possible to limit the access to Secrets using RBAC policies (usually very few roles can access Secrets, and many can access ConfigMaps instead). 
In addition to that, Kubernetes tries to avoid logging secrets and credentials in general

Reply:
@r00ta I'm totally with you on best practice, but I'd say just keep an eye on scope of your PRs. A lot of the changes here are related to your decision to move from `env` vars to a `Secret`, and that adds additional overhead on reviewers to understand that. Two smaller PRs would help the team to review, particularly with our different individual levels of knowledge on K8S.

I'm not suggesting we remove the usage of a `Secret` as you've already done the work and it's great quality. Just something for future consideration.

Reply:
+1